docs/source/releases/1.17.2/1.17.2-bug-use-area-id-vs-description-add-tc-time-to-tc-area-ids.yaml

```diff
@@ -0,0 +1,64 @@
+bug fix:
+  - title: Use area_def.area_id rather than description as the unique identifier
```

Comment:
This update changed how TC sectors are handled,  but didn't make any difference for any other sectors